platform-configs/include/msi-z690-common.robot

```diff
@@ -25,6 +25,7 @@ ${POWER_CTRL}=                                  sonoff
 ${FLASH_VERIFY_METHOD}=                         none
 ${WIFI_CARD}=                                   ${TBD}
 ${MAX_CPU_TEMP}=                                80
+${WDT_DEFAULT}=                                 600
```

Comment:
New variables should be added to defalt.robot, and then to all other configs requiring it.